youtube_dl/extractor/bbc.py

```diff
@@ -796,9 +795,15 @@ def _real_extract(self, url):
                             entries.append(self._extract_from_playlist_sxml(
                                 playlist.get('progressiveDownloadUrl'), playlist_id, timestamp))
 
+        playlist_title = self._og_search_title(webpage, default=None, fatal=False)
+        playlist_title = playlist_title or self._html_search_regex(
+            r'<title>(.*?)(?:\s*-\s*BBC [^ ]+)?</title>', webpage, 'playlist title')
+
+        playlist_title = self._search_regex(r'(.*?) - BBC', playlist_title, 'title')
```

Comment:
Add `default=playlist_title` in order not to fail when `<title>` does not contain ` - BBC`. Regex should match at least one characted and should be greedy `r'(.+)\s*-\s*BBC'`.